testing/test_recwarn.py

```diff
@@ -225,9 +216,6 @@ def test_record(self):
         assert len(record) == 1
         assert str(record[0].message) == "user"
 
-        print(repr(record[0]))
```

Comment:
This was tested the recent change of the `WarningMessage` object to a `namedtuple`, but this is not part of the API and has never been published, so it is safe to go.